src/main/java/spoon/support/reflect/code/CtCatchVariableImpl.java

```diff
@@ -107,6 +107,11 @@ public <C extends CtVariable<T>> C setDefaultExpression(CtExpression<T> defaultE
 		return (C) this;
 	}
 
+	@Override
+	public boolean isUnnamed() {
+		return CtLocalVariableImpl.isUnnamed(this);
```

Comment:
Same here, why is this method on the impl and not the interface?

Reply:
The method holds code that works for all `CtVariable`s in the same way, but isn't really appropriate to be used on e.g. `CtField`. I thought hiding it in one of the implementations would make sense.

Reply:
Okay, I see. Maybe mark the method as `@Internal` then? People will use it if it is public.